src/app/lib/utilities/urlParser/index.js

```diff
@@ -0,0 +1,22 @@
+import Url from 'url-parse';
+
+// Returns the path, excluding any query string params
+export const getUrlPath = url => {
+  return new Url(url).pathname;
```

Comment:
if this every runs in the browser then we might run into problems due to the support of `new Url()` in IE, which I think doesn't support it at all

Reply:
maybe look into https://www.npmjs.com/package/url-polyfill

Reply:
I quickly checked the docs for url-parse -> https://www.npmjs.com/package/url-parse and IE 9/10/11 seems to be OK.  Also, we're only using this override in the test environment for being able to view live data in the test/stage env, so it's probably OK if we don't fully support IE for this functionality...

Reply:
aaah sorry, my mistake. I didn't see the import at the top. I just assumed you were using https://developer.mozilla.org/en-US/docs/Web/API/URL/URL